[2022-01-25 13:46:08,949] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T13:45:41.648219+00:00 [queued]>
[2022-01-25 13:46:08,999] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T13:45:41.648219+00:00 [queued]>
[2022-01-25 13:46:09,000] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 13:46:09,000] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 13:46:09,001] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 13:46:09,048] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): load_data> on 2022-01-25T13:45:41.648219+00:00
[2022-01-25 13:46:09,072] {standard_task_runner.py:52} INFO - Started process 132 to run task
[2022-01-25 13:46:09,091] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'load_data', '2022-01-25T13:45:41.648219+00:00', '--job-id', '159', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpuk0blg0s', '--error-file', '/tmp/tmpm381ee56']
[2022-01-25 13:46:09,121] {standard_task_runner.py:77} INFO - Job 159: Subtask load_data
[2022-01-25 13:46:09,325] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.load_data 2022-01-25T13:45:41.648219+00:00 [running]> on host 05cddda723d4
[2022-01-25 13:46:09,485] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T13:45:41.648219+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T13:45:41.648219+00:00
[2022-01-25 13:46:30,085] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 13:46:30,107] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=load_data, execution_date=20220125T134541, start_date=20220125T134608, end_date=20220125T134630
[2022-01-25 13:46:30,219] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 13:46:30,241] {local_task_job.py:146} INFO - Task exited with return code 0
